webpack/assets/javascripts/react_app/redux/index.js

```diff
@@ -1,12 +1,19 @@
 import createLogger from 'redux-logger';
 import thunk from 'redux-thunk';
+import { routerMiddleware } from 'connected-react-router';
 import { applyMiddleware, createStore, compose } from 'redux';
 
+import history from '../history';
 import reducers from './reducers';
 
 import { IntervalMiddleware, APIMiddleware } from './middlewares';
 
-let middleware = [thunk, IntervalMiddleware, APIMiddleware];
+let middleware = [
+  thunk,
+  IntervalMiddleware,
+  APIMiddleware,
+  routerMiddleware(history),
+];
```

Comment:
just a thought, should we export all middlewares from `'./middlewares/index.js'` ?

Reply:
makes sense